### PR TITLE
New iTwin Viewer starter tutorial

### PR DESCRIPTION
**Pull request by BaoTon**
This adds a new tile and section to the 'Tutorials Page'.

- Renamed 'Building Applications' to 'Starter Applications'.
- New section called 'Building Applications'.

### CODE REVIEW

File: docs/learning/tutorials/index.md [hunk 1]
```diff
@@ -19,13 +19,16 @@ Test iModels can be used for local development. There are a few different ways t
...
 
-### Building Applications
+### Starter Applications
 
 [!bwc tile heading="Web viewer" linkTo="develop-web-viewer" contents="Create a web based viewer" icon="network.svg" step="13" width="28%" ]
 [!bwc tile heading="Desktop viewer" linkTo="develop-desktop-viewer" contents="Create a desktop viewer" icon="computer.svg" step="13" width="28%"]
 [!bwc tile heading="Agent application" linkTo="develop-agent" contents="Create an agent application" icon="notification.svg" step="13" width="28%"]
 [!bwc tile heading="Extension" linkTo="gettingstarted subPath=/frontend/extensions" contents="Create an iTwin.js Extension" icon="puzzle.svg" step="13" width="28%"]
 
+### Building Applications
```
Comment (williamkbentley) [inline docs/learning/tutorials/index.md:29]: Markdown lint: Should have blank line after header

File: docs/learning/tutorials/develop-web-viewer.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
   - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
 - Open the `your-app-name` directory in VS Code.
 - Add a valid contextId (i.e. Project Id) and iModelId for your user to the IMJS_CONTEXT_ID and IMJS_IMODEL_ID variables in the .env file within the application's root directory.
-  - You can obtain these ids from the "Show Ids" column of your [registration dashboard](https://www.itwinjs.org/getting-started/registration-dashboard?tab=1)
+  - You can obtain these ids from the "Show Ids" column of your [registration dashboard]($docs/getting-started/registration-dashboard?tab=1)
```
Comment (williamkbentley) [inline docs/learning/tutorials/develop-web-viewer.md:10]: The query params do not work with $docs syntax

File: docs/learning/tutorials/hello-world-viewer.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+
+This tutorial assumes that you already have:
+
+- Your own local source for the iTwin Web Viewer based on the template @bentley/itwin-viewer
+  - Instructions for that can be found [here]($docs/learning/tutorials/develop-web-viewer/)
+- Configured your local source to open the "House Model" sample iModel.
+  - Instructions to use this sample iModel can be found [here]($docs/learning/tutorials/create-test-imodel-sample/).
+
+![HelloWorldAbove]($docs/learning/tutorials/images/hello_world_above.png)
+### Goal
```
Comment (williamkbentley) [inline docs/learning/tutorials/hello-world-viewer.md:15]: Markdown lint: Blank lines around headers

**Review by williamkbentley — CHANGES_REQUESTED** [review]
Minor markdown lint issues

File: docs/learning/tutorials/develop-web-viewer.md (hunk 2 above)
Comment (BaoTon) [reply docs/learning/tutorials/develop-web-viewer.md]: Fixed

File: docs/learning/tutorials/index.md (hunk 1 above)
Comment (BaoTon) [reply docs/learning/tutorials/index.md:29]: Fixed.

File: docs/learning/tutorials/hello-world-viewer.md (hunk 3 above)
Comment (BaoTon) [reply docs/learning/tutorials/hello-world-viewer.md:16]: Fixed.

**Review by williamkbentley — APPROVED** [review]

File: docs/learning/tutorials/hello-world-viewer.md [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+
+Next switch out the ```helloWidget``` with the new ```backgroundColorWidget``` here:
+
+``` typescript
+    if (location === StagePanelLocation.Right) {
+      const backgroundColorWidget: AbstractWidgetProps = {
+        id: "BackgroundColorWidget",
+        label: "Background Color Toggle",
+        getWidgetContent() {
+          return <Toggle></Toggle>
```
Comment (aruniverse) [inline docs/learning/tutorials/hello-world-viewer.md:179]: nit: shouldn't need the close tag for the toggle

```suggestion
          return <Toggle />
```

Comment (BaoTon) [reply docs/learning/tutorials/hello-world-viewer.md]: Fixed

Comment (aruniverse) [reply docs/learning/tutorials/hello-world-viewer.md]: Could you update the rest of the file to remove the toggle closing tag too? I know its a huge nit pick... sorry!

Comment (BaoTon) [reply docs/learning/tutorials/hello-world-viewer.md]: No worries, Fixed.

**Review by kabentley — APPROVED** [review]

**Pull request merged**